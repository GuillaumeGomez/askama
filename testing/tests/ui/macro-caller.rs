use askama::Template;

#[derive(Template)]
#[template(source = "{%- macro caller() -%}{%- endmacro -%}", ext = "html")]
struct MacroSuper;

fn main() {
}
