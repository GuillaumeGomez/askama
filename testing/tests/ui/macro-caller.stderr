error: 'caller' is not a valid name for a macro
 --> <source attribute>:1:2
       "- macro caller() -%}{%- endmacro -%}"
 --> tests/ui/macro-caller.rs:4:21
  |
4 | #[template(source = "{%- macro caller() -%}{%- endmacro -%}", ext = "html")]
  |                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
